nix/options.nix

```diff
@@ -55,6 +55,14 @@ in
       '';
     };
 
+    deployment.sshOptions = mkOption {
+      type = types.listOf types.str;
+      default = [];
+      description = ''
+        This option specifies extra options passed to OpenSSH.
```

Comment:
```suggestion
        This option specifies extra options passed to the OpenSSH client.
```

Reply:
Is this list interpreted by a shell, or passed via `argv`? Let's clarify that in the option description.

Reply:
```suggestion
        Extra options passed to the OpenSSH client verbatim, and are not executed by a shell.
```